### ISSUE TITLE
[feature] member shortcode for auto renewal status

### ISSUE THREAD

**Issue by jbonlinea** [agileware/wp-civicrm-ux#4]
Hi, 
Thank's for this usefull plug-in.

I'm (re-)digging into your plug-in and think of few functionalities that might be of interest.

Civi provide the possibility to "auto-renew" membership, and I've set up a contribution page to create/renew ones membership which offer this option to the user. 

The member shortcodes are super nice to provide a feedback on front end to the logged in user. 
As far as my little experiment could reveal, the member "expire date" and "renewal date" remain the same whether auto-renewal is on or off. This is what we may expect, no worries, however it would be great if we could have a shortcode that return the "auto-renewal" statut of the member. This would enable to provide a more complete feedback to the user and avoid any confusion.

Regards

**Comment by agileware-justin** [comment]
Yes, the auto-renewal have no effect currently on the expire and renewal dates. Good suggestion @jbonlinea

**Comment by agileware-justin** [comment]
Just noting here that you could solve this problem by setting up a Data Processor for the membership and display any of the membership fields you wanted. Then it's a matter of using the ux_civicrm_listing shortcode to display the results on the page.

**Comment by agileware-justin** [comment]
If you are still interested, recommend checking out the APIv4 shortcode feature, https://github.com/agileware/wp-civicrm-ux/blob/1.11.0/USAGE.md#civicrm-apiv4-shortcode

**Comment by agileware-justin** [comment]
Leaving this open, still a good idea.
